### ISSUE TITLE
Login page: No visible label provided for the dropdown

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4639]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 3.3.2 Labels or instructions.

**Test Environment:**
- Test URL: https://clarity.design/documentation/login
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0

**Steps to reproduce:**
1. Open the test URL and check if visible label is provided for dropdown.

**Expected result:**
Visible label should be provided and placed closely to the dropdown, so speech input users can use the voice command appropriately (assuming accessibile label is same as visible label) and cognitive disability users can understand the purpose of dropdown more clearly. **Reference:** https://www.w3.org/TR/WCAG20-TECHS/G162.html

**Actual result:**
Visible label is not provided for the dropdown.

**Comment by chlane** [comment]
This refers to the actual technique in use. i.e. label element and &quot;for&quot; attribute - https://www.w3.org/WAI/WCAG21/Techniques/html/H44.html. 

This technique requires the labels to be visible. 

See the relevant HTML below.

&lt;clr-select-container class=&quot;clr-form-control&quot;&gt;&lt;label class=&quot;clr-sr-only clr-control-label&quot; for=&quot;clr-form-control-4&quot;&gt;User Role&lt;/label&gt;&lt;!----&gt;&lt;div class=&quot;clr-control-container&quot;&gt;&lt;div class=&quot;clr-select-wrapper&quot;&gt;&lt;select clrselect=&quot;&quot; name=&quot;type&quot; aria-describedby=&quot;clr-form-control-4-helper&quot; id=&quot;clr-form-control-4&quot; class=&quot;clr-select ng-pristine ng-valid ng-touched&quot;&gt;&lt;option value=&quot;local&quot;&gt;Local Users&lt;/option&gt;&lt;option value=&quot;admin&quot;&gt;Administrator&lt;/option&gt;&lt;/select&gt;&lt;!----&gt;&lt;!----&gt;&lt;/div&gt;&lt;!----&gt;&lt;!----&gt;&lt;/div&gt;&lt;/clr-select-container&gt;
